src/main/java/com/glencoesoftware/omero/ms/image/region/ImageRegionMicroserviceVerticle.java

```diff
@@ -191,6 +197,19 @@ public void deploy(JsonObject config, Promise<Void> prom) {
         }
         httpTracing = HttpTracing.newBuilder(tracing).build();
 
+        if(config.containsKey("jmx-metrics-enabled") && config.getBoolean("jmx-metrics-enabled")) {
```

Comment:
Whitespace before `(`, can we just ask for `config.getBoolean()` or will that throw an `NPE`?